File: Dia 17 - Arreglos 2/modificar.c
#include <stdio.h>

int main(void) {
  int arr[] = {25, 50, 75, 100};

  printf("Arrya original\n");
  for(int i = 0; i < 4; i++){
      printf("Elemento en la posición %d: %d\n", i, arr[i]);
  }

  arr[1]=500;

  printf("Arrya Modificado\n");

  for(int i = 0; i < 4; i++){
      printf("Elemento en la posición %d: %d\n", i, arr[i]);
  }
  return 0;
}

File: Dia 17 - Arreglos 2/modificar.py
arr= [1,2,3,4,5]

print("Array Original")
for i in range(0,len(arr)):
  print(f"posicion: {i} valor: {arr[i]}")

arr[3]=100

print("Array Modificado")

for i in range(0,len(arr)):
  print(f"posicion: {i} valor: {arr[i]}")
  